contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -385,11 +385,14 @@ contract VaultHub is PausableUntilWithRoles {
     /// @param _vault The address of the vault
     /// @param _liabilitySharesTarget maximum amount of liabilityShares that will be preserved, the rest will be
     ///         marked as redemptionShares. If value is greater than liabilityShares, redemptionShares are set to 0
-    /// @dev NB: Mechanism to be triggered when Lido Core TVL <= stVaults TVL.
+    /// @dev NB: Mechanism to be triggered when Lido Core TVL <= stVaults TVL
+    /// @dev requires the fresh report
     function setLiabilitySharesTarget(address _vault, uint256 _liabilitySharesTarget) external onlyRole(REDEMPTION_MASTER_ROLE) {
         VaultConnection storage connection = _checkConnection(_vault);
         VaultRecord storage record = _vaultRecord(_vault);
 
+        _requireFreshReport(_vault, record);
```

Comment:
why though?

Reply:
I thought about pausing the vault in an unhealthy state or obligations, but it indeed does not make sense since bringing the report will pause it anyway.